src/components/icon/__snapshots__/icon-unicodes.spec.js.snap

```diff
@@ -0,0 +1,206 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`the unicodes have not changed 1`] = `
+Object {
+  "add": "\\\\e940",
```

Comment:
Is it correct that we have 4 `\` instead of 2?

Reply:
It's adding escape characters which it doesn't seem to do in any of the snapshots for components using `Icon` 🤔

Reply:
So we actually need `"\e940"` `\` by itself is an escape char, so we need a second one to cancel the escape and treat it as a normal backslash.

I'm not sure why jest adds additional slashes, two should be sufficient. My guess is they replace every `\` with two.
I logged out the object to check and the object does only contain two backslashes as expected.

![image](https://user-images.githubusercontent.com/2328042/130758632-507659f3-a87b-4884-b1c3-42c362759e2a.png)